src/test/scala/chiselTests/BundleLiteralSpec.scala

```diff
@@ -28,7 +28,8 @@ class BundleLiteralSpec extends ChiselFlatSpec {
       chisel3.assert(outsideBundleLit.a === 42.U)
       chisel3.assert(outsideBundleLit.b === true.B)
       chisel3.assert(outsideBundleLit.c === MyEnum.sB)
-
+      chisel3.assert(outsideBundleLit.isLit())
+      chisel3.assert(outsideBundleLit.litValue().U === outsideBundleLit.asUInt())
```

Comment:
Is there a literal packing test anywhere? Do we want to test that too, by equivalence with a "magic number" in the test case, instead of solely equivalence with the .asUInt behavior?

Reply:
I think `LiteralExtractorSpec` provides `Lit` test for `Element` type, and tests for `Lit`.
But in my understanding, `Lit` and `LitOption` are two different thing:
`Lit` will return a `Option[BigInt]`, while `LitOption` will return `T` itself.
As for this case:
I think the result of `outsideBundleLit.litValue().U === outsideBundleLit.asUInt()` is only a necessary condition to this test.

If we wanna fully test this, I think the we should test the behavior of firrtl behavior of `chisel3.internal.firrtl.PrimOp.AsUIntOp` is equal to the `(accumulator << width) + masked)` behavior. At my first glance to `firrtl.PrimOps.AsUInt`, this behavior is correct. I think this is good enough here.